samples/serverReports/package.json

```diff
@@ -1,403 +1,408 @@
-								"sizex": 2,
-								"sizey": 1
-							},
-							"widget": {
-								"type-of-contention": {}
-							}
-						},
-						{
-							"name": "Metadata Contention",
-							"gridItemConfig": {
-								"sizex": 2,
-								"sizey": 1
-							},
-							"widget": {
-								"metadata-contention": {}
-							}
-						},
-						{
-							"name": "Allocation Contention",
-							"gridItemConfig": {
-								"sizex": 2,
-								"sizey": 1
-							},
-							"widget": {
-								"allocation-contention": {}
-							}
-						}
-					]
-				}
-			}
-		],
-		"snippets": []
-	},
-	"scripts": {
-		"build": "gulp build",
-		"compile": "gulp compile",
-		"watch": "gulp watch",
-		"postinstall": "node ./node_modules/vscode/bin/install"
-	},
-	"dependencies": {
-		"fs-extra": "^8.1.0",
-		"handlebars": "^4.5.3",
-		"openurl": "^1.1.1"
-	},
-	"devDependencies": {
-		"child-process-promise": "^2.2.1",
-		"del": "^3.0.0",
-		"gulp": "^4.0.0",
-		"gulp-color": "0.0.1",
-		"gulp-sourcemaps": "^2.6.4",
-		"gulp-tslint": "^6.0.2",
-		"gulp-typescript": "^3.2.4",
-		"should": "^13.2.1",
-		"tslint": "^3.14.0",
-		"typemoq": "^2.1.0",
-		"typescript": "^2.9.2",
-		"vsce": "1.36.2",
-		"vscode": "^1.1.6"
-	}
+  "name": "server-report",
+  "displayName": "Server Reports",
+  "description": "Server Reports",
+  "version": "0.4.0",
+  "publisher": "Microsoft",
+  "preview": true,
+  "engines": {
+    "vscode": "^1.26.0",
+    "azdata": "*"
+  },
+  "icon": "images/extension.png",
+  "license": "SEE LICENSE IN LICENSE.txt",
+  "repository": "https://github.com/Microsoft/azuredatastudio",
+  "categories": [
+    "Other"
+  ],
+  "activationEvents": [
+    "*"
+  ],
+  "main": "./out/src/extension",
+  "forceReload": true,
+  "contributes": {
+    "commands": [
+      {
+        "command": "tempdb.startEvent",
+        "title": "Start",
+        "icon": {
+          "light": "./out/src/media/launch.svg",
+          "dark": "./out/src/media/launch_inverse.svg"
+        }
+      },
+      {
+        "command": "tempdb.stopEvent",
+        "title": "Stop",
+        "icon": {
+          "light": "./out/src/media/blocker.svg",
+          "dark": "./out/src/media/blocker_inverse.svg"
+        }
+      },
+      {
+        "command": "tempdb.contention",
+        "title": "Contention Help for Tempdb",
+        "icon": {
+          "light": "./out/src/media/documentation.svg",
+          "dark": "./out/src/media/documentation_inverse.svg"
+        }
+      },
+      {
+        "command": "tempdb.pauseEvent",
+        "title": "Pause",
+        "icon": {
+          "light": "./out/src/media/insights.svg",
+          "dark": "./out/src/media/insights_inverse.svg"
+        }
+      }
+    ],
+    "configuration": [],
+    "views": {},
+    "menus": {},
+    "dashboard.tabs": [
+      {
+        "id": "Server-Reports",
+        "title": "Server Reports",
+        "description": "This extension shows useful reports for a server.",
+        "icon": "images/server_reports.svg",
```

Comment:
like 66 I the only change in this file or more? I see more dev dependencies and such - is that intentional?

Reply:
1. previously this extension is referencing the azdata.d.ts using relative path, i am updating it to use the azdata package (added a devDependencies)
2. i am also moving a internal tab out to the top level tabs, described in the PR description.